evm/validation.py

```diff
@@ -227,3 +227,19 @@ def validate_header_params_for_configuration(header_params):
                 ", ".join(tuple(sorted(extra_fields))),
             )
         )
+
+
+def validate_access_list(access_list):
+    for obj in access_list:
+        if len(obj) == 0:
+            raise ValidationError("Access list entry must at least specify an account address.")
+        address, prefixes = obj[0], obj[1:]
```

Comment:
*fancy python3 stuff nitpick*

We can use the new python 3 star unpacking syntax to do this

```python
address, *prefixes = obj
```